ipapython/version.py.in

```diff
@@ -51,3 +51,7 @@ API_VERSION = "@API_VERSION@"
 DEFAULT_PLUGINS = frozenset(l.strip() for l in """
 @DEFAULT_PLUGINS@
 """.strip().splitlines())
+
+KRB5_VERSION = "@KRB5_VERSION@"
```

Comment:
Probably the naming should reflect the fact this is the krb5 version IPA was built against, not the installed krb5 version...

Reply:
Done.